### DISCUSSION TITLE
Original game text copyright

### DISCUSSION THREAD

**Discussion by a1exsh** [ihhub/fheroes2#4821] [Q&A] [answered]
I've just realized that we include and distribute significant amount of copyrighted material from the original game, namely — all the texts.  Isn't that problematic?..

**Comment by zenseii** [comment]
Maybe the original text could be extracted similar to how the other original assets are gotten?

**Reply by a1exsh** [reply]
I haven't checked it yet, but I guess that these are part of the binary executable file in the OG, so that's going to be quite tricky.

**Comment by ihhub** [comment]
We don't have any text from game resources like maps and the text in fheroes2 is not 100% the same as in the original game.

**Reply by a1exsh** [reply]
Hm, maybe I'm under a false impression, but it felt like some text (e.g. campaign scenario descriptions) are 1:1 copied from OG. Also, it doesn't have to be 100.0% to get into any trouble, IMHO.

I will try playing OG and fheroes2 side-by-side to get a better informed impression.

**Reply by a1exsh** [reply]
Compare, OG:
![heroes2_011](https://user-images.githubusercontent.com/489601/147052236-00ad5c60-ffbe-4277-b88b-2ef8457012ac.png)

fheroes2:
![image](https://user-images.githubusercontent.com/489601/147052258-b8693b68-cd68-4bdf-9f1c-f32486d1f428.png)

**Reply by a1exsh** [reply]
(moved back to the thread):

![heroes2_012](https://user-images.githubusercontent.com/489601/147052608-6fe721ee-8b8d-46c0-8d3e-cd1d535d12ff.png)
![image](https://user-images.githubusercontent.com/489601/147052619-6906fc19-e271-4101-9f30-94a405170066.png)

Is this not 100% the same?..

**Reply by a1exsh** [reply]
![heroes2_014](https://user-images.githubusercontent.com/489601/147053001-d16eb574-aba8-4654-83ed-722ef77cd2cb.png)

![image](https://user-images.githubusercontent.com/489601/147052989-a9b8345c-e9ba-4aad-bc78-eda31aec9d03.png)

**Reply by a1exsh** [reply]
Still 100% the same text:

![heroes2_013](https://user-images.githubusercontent.com/489601/147053224-1f66611c-8953-4cc8-8c6d-c502acf27178.png)

![image](https://user-images.githubusercontent.com/489601/147053188-f8117a4a-03e2-49d1-b98c-97b5a0e4dc60.png)

**Reply by oleg-derevenetz** [reply]
Hi all,

Small texts, names and slogans are not protected by copyright AFAIK. If someone wrote the phrase "I'm having a nice day" it doesn't mean that he can sue any other person who wrote the same phrase. To be eligible for copyright protection, the text must be large enough and at the same time original enough from a literary point of view. It's all pretty shaky.

**Reply by Branikolog** [reply]
I agree with @oleg-derevenetz. Text part of Heroes2 game is not so essential, even from a scenario point, and hardly could be the reason of copyright investigation.
Even so, what can we do here? I'm not sure fans would be glad if we provide campaign scenarios without description.

**Reply by a1exsh** [reply]
Well, I also didn't think this can be problematic until I've seen how much text there actually is (a translation file is close to 9,000 lines).  I'm not suggesting any action at this point, I'm just checking if my understanding is correct: we intend to copy the texts 1:1 and don't expect any problems.  That's fine with me.

**Comment by ihhub** [comment, marked as answer]
I'm recommending to do this way: if someone messes with copyright by adding a text which is a part of game resources then yes, it's a no and we must prevent such. The same for game resources.
As of now I see no issue of keeping really tiny texts which are apparently stored in executable file of the original game as well. Without it the game can't be the same. We can close this discussion for now if everyone agrees with such approach.
